packages/jsii-pacmak/lib/targets/go/types/go-type.ts

```diff
@@ -13,15 +13,15 @@ export abstract class GoType {
   public readonly proxyName: string;
 
   public constructor(public pkg: Package, public type: Type) {
-    this.name = toPascalCase(type.name);
+    this.name = type.name;
```

Comment:
How come?

Reply:
jsii type names (enums/classes/interfaces) are already pascal cased, so there is not reason to screw up with them at this point.